.azure/scripts/run-executable.ps1

```diff
@@ -175,7 +175,7 @@ function PrintDumpCallStack($DumpFile) {
     $env:_NT_SYMBOL_PATH = Split-Path $Path
     try {
         if ($env:BUILD_BUILDNUMBER -ne $null) {
-            $env:PATH += ";c:\Program Files (x86)\Windows Kits\10\Debuggers\x64" 
+            $env:PATH += ";c:\Program Files (x86)\Windows Kits\10\Debuggers\x64"
```

Comment:
I thought we were avoiding hardcoding this?

Reply:
I didn't change this code. This is specific to the AZP machines.